packages/amplication-client/src/Resource/ResourceFormPage.tsx

```diff
@@ -27,6 +30,21 @@ function ResourceFormPage({ match }: Props) {
   const canDeleteResource = permissions.canPerformTask("resource.delete");
   const canEdit = permissions.canPerformTask("resource.*.edit");
 
+  const { data, error, loading } = useQuery<{
+    catalog: {
+      data: models.Resource[];
+      totalCount: number;
+    };
+  }>(RESOURCES_FROM_TEMPLATE_WITH_COUNT, {
+    variables: {
```

Comment:
do not execute the query when the current resource is not a service template - something like 
skip: resource.resourceType !== EnumResourceType.ServiceTemplate